source/_components/cover.rflink.markdown

```diff
@@ -0,0 +1,74 @@
+---
+layout: page
+title: "RFLink Cover"
+description: "Instructions how to integrate RFLink Somfy RTS Cover into Home Assistant."
+date: 2017-08-08
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: rflink.png
+ha_category: Cover
+ha_release: 0.50
+ha_iot_class: "Local Polling"
+---
+
+The `rflink` component support devices that use [RFLink gateway firmware](http://www.nemcon.nl/blog2/), for example the [Nodo RFLink Gateway](https://www.nodo-shop.nl/nl/21-rflink-gateway). RFLink gateway is an Arduino firmware that allows two-way communication with a multitude of RF wireless devices using cheap hardware (Arduino + transceiver).
+
+First you have to set up your [rflink hub](/components/rflink/).
```

Comment:
Not sure what you mean by this comment, since it's a exact copy of  readme from the rflink/light component.
I have rewritten some parts of the readme. So hopefully it's correct now.